Modix.Services/CommandHelp/HiddenAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Modix.Services.CommandHelp
+{
+    /// <summary>
+    /// Hides the module or command from display
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+    public class HiddenAttribute : Attribute
```

Comment:
This should probably have a more specific name that conveys the attribute hides the module from the help command